src/Chirp.Web/Pages/_ViewStart.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@{

```

Comment:
Should the `_ViewStart`-file also be housed in `namespace Chirp.Web.Pages` ? o:

Reply:
This is code copied from the template and seems to work fine as when i comment it out it breaks the login button so I wouldn't worry about it